app/3.0/toolbox/news-case-study.md

```diff
@@ -228,78 +225,14 @@ visits `/list/top_stories`:
   </div>
 </div>
 
-## Resource URLs
```

Comment:
Why are these sections cut?

Reply:
Resource URLs - this seemed server-specific to me. Server-specific config stuff (including app.yaml) was removed in this commit https://github.com/Polymer/news/commit/1a97bb0690eda9bd953eae1bcfdb13f922431b8c

Since the repo is agnostic on how the resource URLs should be handled, I took out the section. Maybe I misunderstood though... wdyt?

The section on AMP I removed because AFAIK there is no AMP branch for 3.0.